src/Sfa.Tl.ResultsAndCertification.Models/Authentication/DfeUserInfo.cs

```diff
@@ -32,5 +32,10 @@ public class DfeUserInfo
         /// <summary>Gets or sets a value indicating whether this instance has access to service.</summary>
         /// <value><c>true</c> if this instance has access to service; otherwise, <c>false</c>.</value>
         public bool HasAccessToService { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets the roles.
+        /// </summary>
+        public string Organisation { get; set; }
```

Comment:
I don't think we need this property. This is only used in the method GetDfeSignInUserInfo to save momentarily the organisation name to be compared in the same method. It is not used anywhere else

Reply:
This will be useful to hold the organisation value.